SECURITY.md

```diff
@@ -0,0 +1,34 @@
+# Security Vulnerability Disclosure and Response Process
+
+As we all know, TiDB is a fast-growing open source database. In order to ensure its security, we have adopted the following security vulnerability disclosure and response process.
+
+The primary goal of this process is to reduce users’ total exposure time to publicly known vulnerabilities.To ensure TiDB products’ vulnerabilities can be fixed quickly, our security team is responsible for the entire vulnerability management process, including internal communication and external disclosure.
+
+If you think you have found a vulnerability or have experienced security incidents involving vulnerabilities of TiDB products, please report it as soon as possible to the TiDB security team (security@tidb.io).
+
+Please kindly help provide as much vulnerability information as possible in the following format:
+
+- Issue title*:
+
+- Overview*:
+
+- Affected components and version number*:
+
+- CVE number (if any):
+
+- Vulnerability verification process*:
+
+- Contact information*:
+
+*mean required
```

Comment:
if it's recommended to file a github issue to report the vulnerability issue, how about direct the audiences to file a security issue?

Reply:
@zz-jason This announcement is to recommend that users submit vulnerabilities in a fixed format using emails, and this announcement will also be published on the official website later. Asktug also explained how to submit vulnerabilities. Guide customers to submit through multiple announcements.